cicd/gitlab/dot.gitlab-ci.yml

```diff
@@ -37,9 +37,7 @@ default:
 
 # Using default stages (https://docs.gitlab.com/ee/ci/yaml/#stages): .pre > build > test > deploy > .post
 
-
```

Comment:
In general, you may want to avoid getting blame from removing empty lines (unless they break compliance with standard)